launch/lidartag_robosense_m1.launch.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * Copyright (C) 2013-2025, The Regents of The University of Michigan.
+ * All rights reserved.
+ * This software was developed in the Biped Lab (https://www.biped.solutions/)
+ * under the direction of Jessy Grizzle, grizzle@umich.edu. This software may
+ * be available under alternative licensing terms; contact the address above.
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ * The views and conclusions contained in the software and documentation are those
+ * of the authors and should not be interpreted as representing official policies,
+ * either expressed or implied, of the Regents of The University of Michigan.
+ *
+ * AUTHOR: Bruce JK Huang (bjhuang[at]umich.edu)
+ * WEBSITE: https://www.brucerobot.com/
+-->
+
+<launch>
+
+  <arg name="pointcloud_topic" default="/pointcloud_raw" />
+  <arg name="use_intensity_channel" default="true" />
+
+  <include file="$(find-pkg-share lidartag)/launch/lidartag_master.launch.xml" >
+
+    <arg name="pointcloud_topic" value="$(var pointcloud_topic)" />
+    <arg name="use_intensity_channel" value="$(var use_intensity_channel)" />
+    <arg name="beam_number" value="128" />
+    <arg name="has_ring" value="false" />
+    <arg name="horizontal_fov" value="120.0" /> <!-- Comment: This is solid state lidar with a reduced fov compared with traditional mechanical lidars https://www.hesaitech.com/product/at128/ -->
```

Comment:
Pedantic, but let's remove the reference to AT128 since this is for Robosense :)

Reply:
Addressed in [82c47e9](https://github.com/tier4/LiDARTag/pull/20/commits/82c47e9c3c1022c4aee72ba5cfb3e79e826ed9ac)
Forgot to mark it as such, but this is a draft (until we do a proper calibration with a lidartag mount)